draft-ietf-rtcweb-jsep.xml

```diff
@@ -2824,6 +2824,35 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             with CN codecs and "usedtx=0").</t>
           </section>
         </section>
+        <section title="Direction Attribute in Answers"
+        anchor="sec.answer-dirattr">
+
+          <t><xref target="RFC3264"></xref> direction attributes
+          (defined in Section 6.1) in answers are chosen according to
+          the direction attribute in the remote offer and the direction
+          of the corresponding RtpTransceiver, by choosing the
+          intersection of the two directions in the following table:</t>
```

Comment:
I had always assumed the algorithm was as follows:

enum {
  inactive = 0,
  sendonly = 1,
  recevonly = 2,
  sendrecv =3
};

int transceiver;   // How the transceiver is set
int offer;             // How the offer was set

int offer_swapped = (offer & 1) << 1  | (offer & 2) >> 1
answer = offer_swapped & transceiver;

Is this right?